tests/stubtest_allowlists/py3_common.txt

```diff
@@ -234,6 +237,14 @@ builtins.reveal_locals
 builtins.reveal_type
 # GetSetDescriptor that always raises AttributeError
 builtins.OSError.characters_written
+builtins.EnvironmentError.characters_written
+builtins.IOError.characters_written
```

Comment:
```suggestion
builtins.(OS|Environment|IO)Error.characters_written
```

Reply:
I don't really like this, as it makes the errors less greppable.

Reply:
OK. I do think there should be a comment to the effect that these are all, actually, aliases for `OSError`, though.